client/my-sites/checkout/src/components/akismet-pro-quantity-dropdown/index.tsx

```diff
@@ -110,30 +116,103 @@ export const AkismetProQuantityDropDown: FunctionComponent< AkismetProQuantityDr
 		responseCart.products[ 0 ].quantity ?? 1
 	);
 
-	const dropdownOptions = [ 1, 2, 3, 4 ];
+	const dropdownOptions = useMemo(
+		() => [
+			{
+				id: 1,
+				label: translate( '1 Site' ),
+			},
+			{
+				id: 2,
+				label: translate( '2 Sites' ),
+			},
+			{
+				id: 3,
+				label: translate( '3 Sites' ),
+			},
+			{
+				id: 4,
+				label: translate( '4 Sites' ),
+			},
+			{
+				id: 5,
+				label: translate( 'Unlimited sites (Akismet Business)' ),
+			},
+		],
+		[]
+	);
+
+	const AK_BUSINESS_DROPDOWN_ID = dropdownOptions[ dropdownOptions.length - 1 ].id;
```

Comment:
Maybe we can also keep that variable in `useMemo`? I'm not sure, but this calculation might trigger unnecessary re-renders + it uses only values from the previous useMemo.

Also, uppercase here might confuse as it's not static constant - it might change in the runtime.